### PR TITLE
Support null-value deletes 

### PR DESCRIPTION
**Pull request by ableegoldman**
Both KV and Window stores allow deletion via put with a null value in Kafka Streams. We need to handle these properly as tombstones and enable deletes for window stores

### CODE REVIEW

File: kafka-client/src/main/java/dev/responsive/db/CassandraWindowedSchema.java [hunk 1]
```diff
@@ -98,7 +100,7 @@ public CassandraWindowedSchema(final CassandraClient client) {
 
   @Override
   public void create(final String name, Optional<Duration> ttl) {
-    // TODO: explore better data models for fetchRange/fetchAll
+    // TODO(window): explore better data models for fetchRange/fetchAll
```
Comment (rodesai) [inline kafka-client/src/main/java/dev/responsive/db/CassandraWindowedSchema.java:103]: ultra-nit: usually this notation is used to make note of who owns the todo (e.g. TODO(sophie))

**Review by rodesai — COMMENTED** [review]
The code change LGTM. We should add unit tests for this stuff or its going to be hard to confidently refactor/change things later. Definitely don't want to block this change on that though.

File: kafka-client/src/main/java/dev/responsive/db/CassandraWindowedSchema.java (hunk 1 above)
Comment (ableegoldman) [reply kafka-client/src/main/java/dev/responsive/db/CassandraWindowedSchema.java:103]: yeah, I guess it's technically targeted at myself but I just wanted to mark everything I want to hit in the next window store PR. so it should be very temporary

**Comment by ableegoldman** [comment]
Thanks, I'm going to merge it for now but totally agree on the tests. I was hoping to just throw in a quick unit test for this but found out we don't really have a unit test framework for the state stores yet, so it's kind of a bigger task than I wanted to take on just yet. I'll file a ticket for that though

**Pull request merged**